src/Microsoft.Android.Templates/android-bindinglib/.template.config/template.json

```diff
@@ -1,10 +1,10 @@
 {
   "$schema": "http://json.schemastore.org/template",
   "author": "Microsoft",
-  "classifications": [ "Android" ],
+  "classifications": [ "Android", "Mobile" ],
   "identity": "Microsoft.Android.AndroidBinding",
-  "name": "Android Java Library Binding",
-  "description": "A project for creating an Android class library that binds to a native Java library",
+  "name": "Android Java Library Binding (Preview)",
+  "description": "A project for creating a .NET 6 Android class library that binds to a native Java library",
```

Comment:
Will we have to change this to say `.NET 7` one day?

If you have 6.0.100 workloads and 7.0.100 workloads, will both templates show up? I guess so?

Reply:
Hopefully we'll remember to update this when changing  `TargetFramework`s  https://github.com/xamarin/xamarin-android/blob/da5b904e188e848180ab603d2f75b34deb2bd485//src/Microsoft.Android.Templates/android/AndroidApp1.csproj#L3 in the future.

Good question about SxS workload manifests, I am assuming that `dotnet new -l` would only include the template packs that are declared in the workload manifests for the SDK band being used. I guess we'll start to find out some of that stuff once we have a 6.0.200 or 7.0.100 SDK to test against.